packages/reqresp/src/protocols/LightClientFinalityUpdate.ts

```diff
@@ -15,5 +15,17 @@ export const LightClientFinalityUpdate: ProtocolDefinitionGenerator<null, altair
     requestType: () => null,
     responseType: () => ssz.altair.LightClientFinalityUpdate,
     contextBytes: getContextBytesLightclient((update) => modules.config.getForkName(update.signatureSlot), modules),
+    inboundRateLimits: {
+      /**
+       * Finality updates can't be passed more frequently than once per epoch.
+       * So for one peer we allow more relaxed double.
+       *
+       * 384 seconds is chosen to be fair and equivalent to 1 epoch. Can be updated in future.
+       *
+       * For total we multiply with `10` to have lower peer count on light client.
+       */
+      byPeer: {quota: 2, quotaTime: seconds(384)},
```

Comment:
This limits are very restrictive, it's almost free to use to produce this updates. Just use 2 / 12sec like for optimistic